app/vite.config.mts

```diff
@@ -5,17 +5,16 @@
 import { lezer } from "@lezer/generator/rollup";
 import react from "@vitejs/plugin-react";
 import { resolve } from "path";
-import { visualizer } from "rollup-plugin-visualizer";
+// Uncomment below to visualize the bundle size after running the build command, also uncomment plugins.push(visualizer());
+// import { visualizer } from "rollup-plugin-visualizer";
 /// <reference types="vitest/config" />
 import { defineConfig } from "vite";
 import relay from "vite-plugin-relay";
 
-export default defineConfig(({ command }) => {
+export default defineConfig(() => {
   const plugins = [react(), relay, lezer()];
-  // Development uses the serve command
-  if (command === "serve") {
```

Comment:
the serve command doesn't use this anyways it only happens with the build command so this did nothing, switched to comment, can remove if we don't want that though it's nice to have for testing